src/Symfony/Bundle/TwigBundle/DependencyInjection/TwigExtension.php

```diff
@@ -122,6 +122,17 @@ public function load(array $configs, ContainerBuilder $container)
             $twigFilesystemLoaderDefinition->addMethodCall('addPath', array($dir));
         }
 
+        if ($container->fileExists($dir = $container->getParameter('kernel.project_dir').'/templates', false)) {
+            $twigFilesystemLoaderDefinition->addMethodCall('addPath', array($dir));
+        }
+
+        if ($container->fileExists($dir = $container->getParameter('kernel.project_dir').'/templates/bundles', false)) {
+            foreach (glob($dir.'/*', GLOB_ONLYDIR) as $path) {
```

Comment:
Btw previously the Templating component were resolving templates, now we just use (support) Twig namespaces.